rakeshnrao-san_ramon.md

```diff
@@ -0,0 +1,6 @@
+# San Ramon, CA
+
+### Land of earth and wind
+
+### Pleanty of open space
+
```

Comment:
Trying to add a line comment?  Code Comment?  Not sure what this is called.  but I am trying to add something in here